app/templates/save-and-exit.html

```diff
@@ -0,0 +1,20 @@
+{% if display_region == "ni" %}
+    {% extends "base-ni.html" %}
+    {% set start_link = url('Index:get') %}
+{% elif display_region == "cy" %}
+    {% extends "base-cy.html" %}
+    {% set start_link = url('Index:get') %}
+{% else %}
+    {% extends "base-en.html" %}
+    {% set start_link = url('Index:get') %}
```

Comment:
These are all identical. Is this right?

Reply:
This is an error - they should be different, for each display_region - should be resolved now